apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Benchmarks/ValueSerializerBenchmark.cs

```diff
@@ -20,36 +20,38 @@ namespace Google.Cloud.Firestore.Benchmarks
 {
     public class ValueSerializerBenchmark
     {
+        private static readonly SerializationContext _serializationContext = new SerializationContext(null);
```

Comment:
`= SerializationContext.Default` ?

Reply:
Good idea. I probably hadn't created SerializationContext.Default when I made that change :)